manifests/install.pp

```diff
@@ -3,5 +3,14 @@
 class dns::install {
   if ! empty($dns::dns_server_package) {
     ensure_packages([$dns::dns_server_package])
+    $pkg_req = Package[$dns::dns_server_package]
+  } else {
+    $pkg_req = undef
+  }
+
+  if $dns::group_manage {
+    group { $dns::params::group:
```

Comment:
This was already incorrect previously, but mind fixing it while you're at it?
```suggestion
    group { $dns::group:
```

Reply:
Fixed